Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs

```diff
@@ -0,0 +1,73 @@
+using System.Net;
+using Allure.Commons;
+using FluentAssertions;
+using NUnit.Allure.Attributes;
+using NUnit.Allure.Core;
+using NUnit.Framework;
+using Qase_Test.Core;
+using Qase_Test.Fakers;
+using Qase_Test.Models;
+using Qase_Test.Steps.ApiSteps;
+
+namespace Qase_Test.Tests.ApiTests
+{
+    [AllureNUnit]
+    [AllureTag("Functional")]
+    [AllureSeverity(SeverityLevel.critical)]
+    [AllureOwner("Bachurin")]
+    [AllureSuite("API")]
+    public class ProjectApiTests
+    {
+        private Project _project;
+        private string _token;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _project = TestDataGeneratorService.GetFakeProject();
+            _token = UserSettings.Token;
```

Comment:
why can't we keep the token in client, it's always the same, but the tests in the class are reading the settings file every single run in the suite